neusomatic/python/preprocess.py

```diff
@@ -21,19 +21,22 @@
 from utils import concatenate_vcfs
 
 FORMAT = '%(levelname)s %(asctime)-15s %(name)-20s %(message)s'
-logFormatter = logging.Formatter(FORMAT)
+logging.basicConfig(level=logging.INFO, format=FORMAT)
 logger = logging.getLogger(__name__)
```

Comment:
Since only functions, do not need to define file-specific `logger`. We should function-specific `logger` variable.